### PR TITLE
docs: add more commands to cli readme

### PR DESCRIPTION
**Pull request by nabdelgadir**
Connect to https://github.com/strongloop/loopback-next/issues/2269.

Add `lb4 respository`. `lb4 service`, and `lb4 openapi` to CLI package's README.

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

**Comment by nabdelgadir** [comment]
@slnode test please

**Comment by bajtos** [comment]
At the moment, the code blocks are indented at root level, they are not nested inside the list item they belong to. Should we fix that problem while making changes in README?

**Comment by nabdelgadir** [comment]
@bajtos Yes, that's a good idea. I'll update this README.

File: packages/cli/README.md [hunk 1]
```diff
@@ -14,150 +14,208 @@ Run the following command to install the CLI.
...
+            --id             # A valid ID property name for the specified model
+            --datasource     # A valid datasource name
+
+    Arguments:
+      name  # Name for the repository   Type: String  Required: false
+    ```
+
+7.  To scaffold a Service into your application
+
+    `lb4 service`
```
Comment (jannyHou) [inline packages/cli/README.md]: Nitpick: I like the hint of going into the project directory in the doc for `lb4 controller`, could you add it for the new commends as well?

```
    cd <your-project-directory>
```

Same comment apply to the other commends.

**Pull request merged**